packages/theme-chalk/src/popconfirm.scss

```
@@ -5,10 +5,13 @@
   outline: none;
   @include e(main) {
     display: flex;
-    align-items: center;
+    align-items: flex-start;
   }
   @include e(icon) {
     margin-right: 5px;
+    flex-shrink: 0;
+    // Keep the icon optically aligned with the first line of text
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)